test/runtests.jl

```diff
@@ -16,3 +18,74 @@ using Test
         PortAudio.devices()
     end
 end
+
+if !isempty(PortAudio.devices())
```

Comment:
Should we have an else condition so that it errors if no devices are found? For example, if github change the CI runners and they no longer have audio devices.

Reply:
Hmm, well, an audio device is currently only available on mac, so we could potentially error there, but if this section doesn't run the only consequence will be a drop in coverage. Alternatively, we could try to figure out how to simulate audio devices on all platforms. I guess jack can work on linux. I've tried and failed to get jack built with BinaryBuilder, ref https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/863